docs/adr/000x-modelling-history.md

```diff
@@ -0,0 +1,250 @@
+# X. Modelling history
+
+Date: 2018-12-17  
+Amended: 2019-01-29
+
+## Context
+
+The Content Publisher domain model is focused on storing current
+information. There is a documents table which stores the most recent content of
+a document and an images table which stores the most recent version of
+a document's images. History of these changes is stored using [Papertrail][],
+which is not [intended as a permanent store][papertrail-pr]. The presentation
+of a documents history is done via a TimelineEntry model which stores that a
+user did something, but not what they did. In places where richer information
+was required there have been models such as Removal or Withdrawal associated
+with a TimelineEntry.
+
+This has led to a number of pain points:
+
+- users cannot discard a draft of a document that has between previously
+  published because there isn't a data representation it can be reverted to;
+- Content Publiser can't show an accurate link or status for the live edition
+  of a document;
+- users cannot edit or remove images on a document once the first
+  edition is published;
+- the TimelineEntry model stores aspects of a documents state, resulting in it
+  needed to be queried outside a timeline context.
+
+And this prevents a number of intended features for Content Publisher:
+
+- comparing different editions of a document;
+- republishing live content if there is any problems - which is a frequent
```

Comment:
Suggested rephrasing:
"republishing live content if there are any problems (currently a common support task for Whitehall publisher)"